assets/js/components.js

```diff
@@ -57,6 +57,19 @@ function accordion($el) {
   return new Accordion($el);
 }
 
+function togglePassword($el) {
+  var fieldSelector =  '#' + $el.attr('aria-controls'),
```

Comment:
The selector should end up as `#password` as aria-controls = `password`. What aspect isn't working? I'm expecting it to change the password field to a normal text field and back to show and hide what you typed into it.